client/my-sites/importer/newsletter/subscribers/paid-subscribers/connect-stripe.tsx

```diff
@@ -50,7 +50,7 @@ export default function ConnectStripe( {
 			<p>
 				{ createInterpolateElement(
 					__(
-						"To migrate your <strong>paid subscribers</strong> to WordPress.com, make sure you're connecting the <strong>same</strong> Stripe account you use with Substack."
```

Comment:
@crisbusquets I could remove the WordPress.com bit just for Jetpack sites, or all sites for simplicity. WDYT? We could also leave it in of course but felt a bit odd in Jetpack site context.